.docs/summary-generator.cjs

```diff
@@ -18,7 +17,9 @@ function capitalizeFirstLetter(string) {
 }
 
 (async () => {
-  const logStream = fs.createWriteStream(path.join(directoryPath, ".SUMMARY.md"), { flags: "w" });
+  const docsDir = process.argv[2] || "docs";
+  const directoryPath = path.join(__dirname, "..", docsDir);
+  const logStream = fs.createWriteStream(path.join(process.argv[3], "overview.md"), { flags: "w" });
```

Comment:
I get an error: `The "path" argument must be of type string. Received undefined` if i run it withouts args: `npm run docs:md`. Because `process.argv[3]` is undefined and must be a string. here I think it should be: `path.join(process.argv[3] || '"", "overview.md")` or rather `path.join(directoryPath, "overview.md")` .. ?